velero-plugins/service/restore_test.go

```diff
@@ -0,0 +1,65 @@
+package service
+
+import (
+	"encoding/json"
+	"github.com/konveyor/openshift-velero-plugin/velero-plugins/util/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
+	corev1API "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"reflect"
+	"testing"
+)
+
+func TestRestorePluginAppliesTo(t *testing.T) {
+	restorePlugin := &RestorePlugin{Log: test.NewLogger()}
+	actual, err := restorePlugin.AppliesTo()
+	require.NoError(t, err)
+	assert.Equal(t, velero.ResourceSelector{IncludedResources: []string{"services"}}, actual)
+}
+
+func TestRestorePlugin_Execute(t *testing.T) {
+	restorePlugin := &RestorePlugin{Log: test.NewLogger()}
+
+	testcase := map[string]struct {
+		service corev1API.Service
+		want    []string
+	}{
+		"WithoutLoadBalancer": {service: corev1API.Service{Spec: corev1API.ServiceSpec{
+			Type:        "WithoutLoadBalancer",
+			ExternalIPs: []string{"test"},
```

Comment:
Maybe use a fake IP address as the string :)

Reply:
changed to using a fake ip address